manual_correction.py

```diff
@@ -386,6 +398,17 @@ def correct_pmj_label(fname, fname_label, viewer='sct_label_utils'):
         viewer_not_found(viewer)
 
 
+def correct_centerline(fname, fname_label, viewer='sct_label_utils'):
+    """
+    Open sct_label_utils to manually label centerline.
```

Comment:
Are you sure this `sct_label_utils` ? Asking because the command run below is `sct_get_centerline`, so it's a bit confusing

Reply:
Good catch again! Thanks! Fixed in https://github.com/spinalcordtoolbox/manual-correction/pull/33/commits/a800e235bacfeb0744d85a0cde1f4c07b6aa7e36